checkers/replication.py

```diff
@@ -3,7 +3,7 @@
 import os
 import time
 import logging
-
+from pprint import pformat
 
```

Comment:
Missing new line before class name

Reply:
👍 I thought it was better to keep imports in the same block.

Reply:
Yes it is, you need two empty lines between the last import and the class declaration.

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Ops! Didn't knew it. Thanks for the reference link.